service/resource/legacy/load_balancer.go

```diff
@@ -35,7 +35,7 @@ type LoadBalancerInput struct {
 }
 
 func (s *Resource) createLoadBalancer(input LoadBalancerInput) (*awsresources.ELB, error) {
-	lbName, err := loadBalancerName(input.Name, input.Cluster)
+	lbName, err := cloudformation.LoadBalancerName(input.Name, input.Cluster)
```

Comment:
If we move LoadBalancerName to the key package I'd be fine with updating it here. Although IMO leaving the logic as is would also be OK.

Reply:
Yep, moved to the key package.